packages/lexical/src/nodes/LexicalTextNode.ts

```diff
@@ -473,6 +473,13 @@ export class TextNode extends LexicalNode {
     return true;
   }
 
+  /**
+   * @returns true if the text node is inline, false otherwise.
+   */
+  isInline(): boolean {
```

Comment:
If this does make sense to implement on TextNode, it would make sense to also narrow the implementation to return only true, because it's not permitted for any subclass to return false

```suggestion
  isInline(): true {
```